apps/staking/app/address/page.tsx

```diff
@@ -0,0 +1,9 @@
+'use client';
+
+import AddressPage from '@/app/address/[address]/page';
+import { useWallet } from '@session/wallet/hooks/wallet-hooks';
+
+export default function Page() {
+  const { address } = useWallet();
+  return address ? <AddressPage params={{ address }} /> : null;
```

Comment:
Should this not do display a not found page instead?

Reply:
Also wild that you have to do this. I would have thought next was smart enough to catch the root and forward to down since you are using an [address] slug.

Reply:
This is outside the [address] dir, so isn't given an address, without this page we get a 404 as there's no route matching just `/address` only `/address/0x...`.
So now when you go to `/address` you see the view for the connected wallet, otherwise nothing. Will eventually have a search box but that's very much a nice to have.

Reply:
Cool